src/Toggle/Object/GetAllToggleStatusRequest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Clearbooks\Labs\Toggle\Object;
+
+use Clearbooks\Labs\Client\Toggle\Entity\Group;
+use Clearbooks\Labs\Client\Toggle\Entity\Segment;
+use Clearbooks\Labs\Client\Toggle\Entity\User;
+
+class GetAllToggleStatusRequest
+{
+    /**
+     * @var User
+     */
+    private $user;
+
+    /**
+     * @var Group
+     */
+    private $group;
+
+    /**
+     * @var Segment[]
+     */
+    private $segments;
+
+    /**
+     * @param User $user
+     * @param Group $group
+     * @param Segment[] $segments
+     */
+    public function __construct( User $user, Group $group, array $segments )
```

Comment:
Well it's a complex decision. The logic for that is here if you're interested:
https://github.com/clearbooks/labs-php-client/blob/master/src/Toggle/StatelessToggleChecker.php#L176

We pass the current user and group identity and all the segments which this current user-group pair is part of. Then the class above decides if based on these the toggle is active or not :)